packages/ast-spec/tests/fixtures-with-differences-errors.shot

```diff
@@ -17,6 +17,14 @@ exports[`AST Fixtures List fixtures with Error differences 1`] = `
     "declaration/TSInterfaceDeclaration/fixtures/_error_/missing-type-param/fixture.ts",
     "declaration/TSInterfaceDeclaration/fixtures/_error_/non-identifier-extends/fixture.ts",
     "declaration/TSTypeAliasDeclaration/fixtures/_error_/missing-type-parameter/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/const-destructure-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/const-destructure-type-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/const-id-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/const-id-type-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/let-destructure-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/let-destructure-type-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/var-destructure-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/var-destructure-type-no-init/fixture.ts",
```

Comment:
These are because we don't have the ability to detect whether a declaration is in an ambient context unless itself has `declare`